website/project/views/node.py

```diff
@@ -830,6 +832,15 @@ def get_affiliated_institutions(obj):
         })
     return ret
 
+def serialize_collections(cgms):
+    return [{
+        'title': cgm.collection.title,
```

Comment:
Consider also including the collection's provider's name / branding. The mock-up had provider icons, but I don't know if we have a good system set up for that yet.